transport/round_trippers.go

```diff
@@ -434,7 +434,7 @@ func shouldEscape(b byte) bool {
 }
 
 func headerKeyEscape(key string) string {
-	buf := strings.Builder{}
+	buf := bytes.Buffer{}
```

Comment:
`strings.Builder` is more efficient than `bytes.Buffer{}`. I'm not sure if we'll want to change this because kubernetes uses Go 1.11.2 right now.